src/node_contextify.cc

```diff
@@ -125,7 +125,14 @@ class ContextifyContext {
     int length = names->Length();
     for (int i = 0; i < length; i++) {
       Local<String> key = names->Get(i)->ToString(env()->isolate());
-      bool has = sandbox_obj->HasOwnProperty(context, key).FromJust();
+      auto maybe_has = sandbox_obj->HasOwnProperty(context, key);
+
+      // Check for pending exceptions
+      if (!maybe_has.IsJust())
+        break;
+
+      auto has = maybe_has.FromJust();
```

Comment:
tiny nit: I would be okay with just keeping the `bool` type explicit, using `auto` here doesn’t save any space and erases information that a reader of the code may find helpful.